src/Verismith.hs

```diff
@@ -3,7 +3,7 @@
 -- |
 -- Module      : Verismith
 -- Description : Verismith
--- Copyright   : (c) 2018-2019, Yann Herklotz
+-- Copyright   : (c) 2018-2022, Yann Herklotz
```

Comment:
You can add your name there too

Reply:
I might do it in the future, not for changes this small though.